crates/atuin/src/shell/atuin.bash

```diff
@@ -263,6 +264,17 @@ __atuin_history() {
 
         READLINE_LINE=""
         READLINE_POINT=${#READLINE_LINE}
+    elif [[ $__atuin_output == __atuin_edit_at__:* ]]; then
+        # Edit selected command at specified cursor position
+        __atuin_output=${__atuin_output#__atuin_edit_at__:}
+        local __atuin_edit_at_rx='^([0-9]+):(.*)$'
+        if [[ $__atuin_output =~ $__atuin_edit_at_rx ]]; then
+            READLINE_LINE="${BASH_REMATCH[2]}"
+            READLINE_POINT=${BASH_REMATCH[1]}
+        else
+            READLINE_LINE=$__atuin_output
+            READLINE_POINT=${#READLINE_LINE}
+        fi
```

Comment:
It could probably simplified as

```suggestion
    elif [[ $__atuin_output == $__atuin_edit_at_rx ]]; then
        # Edit selected command at specified cursor position
        READLINE_LINE="${BASH_REMATCH[2]}"
        READLINE_POINT=${BASH_REMATCH[1]}
```

with

```bash
local __atuin_edit_at_rx='^__atuin_edit_at:([0-9]+):(.*)$'
```

defined earlier.

Reply:
Indeed, just with =~.